.sonarqube/sonar-scanner.properties

```diff
@@ -46,7 +46,6 @@ sonar.modules= bsm_generator, \
   carma_wm, \
   carma_wm_ctrl, \
   roadway_objects, \
-  platooning_strategic_IHP, \
```

Comment:
Should this be replaced with platooning_strategic_ihp?

Reply:
goto catch, added